### ISSUE TITLE
Logs not displaying detailed view

### ISSUE THREAD

**Issue by IgorCattusso** [IgorCattusso/routing#8]
### Issue

Log details are not being displayed due to an error to parse the JSON returned by the Flask application due to a None type data in the JSON.

```JavaScript
VM4940:1 Uncaught SyntaxError: Unexpected token 'N', ..."ag_pais": None, "mes"... is not valid JSON
    at JSON.parse (<anonymous>)
    at logs.js:5:31
    at HTMLCollection.forEach (<anonymous>)
    at formatJSON (logs.js:3:29)
    at logs.js:11:1
(anonymous) @ logs.js:5
formatJSON @ logs.js:3
(anonymous) @ logs.js:11
```

**Comment by IgorCattusso** [comment]
Fixed! Waiting for commit to resolve the issue

**Comment by IgorCattusso** [comment]
Fixed on Commit #4b9a852
